File: Язык программирования С++/Условный оператор/Алгебра и геометрия/I.cpp
#include <bits/stdc++.h>
using namespace std;
int main() {
  int l1, w1, h1, l2, w2, h2, lc, wc, hc, l3, w3;
  cin >> l1 >> w1 >> h1;
  cin >> l2 >> w2 >> h2;
  cin >> lc >> wc >> hc;
  string f = "NO";
  if (l1 > w1) swap(l1, w1);
  if (l2 > w2) swap(l2, w2);
  if (lc > wc) swap(lc, wc);
  if (h1 + h2 <= hc) {
    if (max(l1, l2) <= lc && max(w1, w2) <= wc)
      f = "YES";
  }
  if (max(h1, h2) <= hc) {
    l3 = l1 + l2;
    w3 = max(w1, w2);
    if (l3 > w3) swap(l3, w3);
    if (l3 <= lc && w3 <= wc)
      f = "YES";
  }
  cout << f;
  return 0;
}

File: Язык программирования С++/Условный оператор/Алгебра и геометрия/I.py
l1, w1, h1 = map(int, input().split())
l2, w2, h2 = map(int, input().split())
lc, wc, hc = map(int, input().split())
if l1 > w1: l1, w1 = w1, l1
if l2 > w2: l2, w2 = w2, l2
if lc > wc: lc, wc = wc, lc
f = "NO"
if h1 + h2 <= hc:
  if max(l1, l2) <= lc and max(w1, w2) <= wc:
    f = "YES"
if max(h1, h2) <= hc:
  l3 = l1 + l2
  w3 = max(w1, w2)
  if l3 > w3: l3, w3 = w3, l3
  if l3 <= lc and w3 <= wc:
    f = "YES"
print(f)